app/views/account/tenants/_tenants_table.html.slim

```diff
@@ -0,0 +1,15 @@
+table.table.table-bordered
+  thead
+    th.col-md-1 ID
+    th.col-md-3 Name
+    th.col-md-2 Website
+    th.col-md-1 Users
+    th.col-md-3 Added
+  tbody
+    - tenants.each do |tenant|
+      tr
+        td= tenant.id
+        td= link_to span = tenant.name, account_tenant_path(tenant)
```

Comment:
error is here `link_to span = tenant.name, account_tenant_path(tenant)`
so now
add spec for INDEX action and make sure tenant is rendered on the page